can_dump_v2.go

```diff
@@ -40,7 +41,7 @@ func (p *canDumpV2Cmd) Execute(_ context.Context, _ *flag.FlagSet, _ ...interfac
 
 	sck, err := canbus.New()
 	if err != nil {
-		p.logger.Fatal().Err(err).Msg("failed to connect to CAN")
+		hooks.LogFatal(p.logger, err, "failed to connect to CAN")
```

Comment:
does this mean we log to edge-logs or not. For this case, since this is a CLI command I don't really care about logging anything from it to edge-logs

Reply:
yes, it means that it will be  logged to edge-logs. All Fatals we were logging.